service/node.go

```diff
@@ -420,6 +428,33 @@ func (s *service) nodeProbe(ctx context.Context) error {
 	return nil
 }
 
+func (s *service) approveSDC(opts Opts) error {
+
+	for _, systemID := range connectedSystemID {
+		system := s.systems[systemID]
+
+		//fetch SDC details
+		sdc, err := s.systems[systemID].FindSdc("SdcGUID", opts.SdcGUID)
+		if err != nil {
+			return status.Errorf(codes.FailedPrecondition, "%s", err)
+		}
+
+		//fetch the restrictedSdcMode
+		if system.System.RestrictedSdcModeEnabled && system.System.RestrictedSdcMode == "Guid" {
```

Comment:
Could we just check  system.System.RestrictedSdcMode here? 
I thought it was sent to None if the mode wasn't enabled. If I'm not remembering right, ignore this comment.

Reply:
yes you are right. if the RestrictedSdcModeEnabled is false the RestrictedSdcMode is set to "None".  I will update it.